### DISCUSSION TITLE
Conversion Error

### DISCUSSION THREAD

**Discussion by Joao-Victor-Liporini** [Kas-tle/java2bedrock.sh#9152] [Converter Help]
Hello, How's it going?

I tried to convert the resource pack of my minecraft server. The only problem is that after doing so, it didn't work.

There were no textures, the custom mapping was blank, as was pretty much everything else.

I'm not sure if this is a problem with my server's  [pack](https://cdn.discordapp.com/attachments/1116760012166017044/1128801533195210803/generated_1.zip) or with the converter itself, but regardless, can you try to help me solve the problem?

Comments: In my texture contains items from itemsadder model engine if I'm not mistaken, items for optifine and most important of all my server tags with the Chinese characters, I really wanted it to work as my server is based on those characters

I tried to convert several times never worked

#9151 
#7572 
#7573
 
And several more times.

![ItemsAdder Items](https://github.com/Kas-tle/java2bedrock.sh/assets/81539403/197b8fd7-315a-41c0-b08e-05ed0d24b849)
ItemsAdder Items

![Chinese Character Tags and Icons](https://github.com/Kas-tle/java2bedrock.sh/assets/81539403/193c07d0-717e-4136-8f3a-1b9061879fab)
Chinese Character Tags and Icons

Thanks in advance, JVLip13

**Comment by typicalsmc** [comment]
the script is unable to convert "emojis" at the moment, can you try checking the errrors 
https://github.com/Kas-tle/java2bedrock.sh/actions/runs/5536662000/jobs/10104614158#step:6:5849
the script and geyser does not support custom blocks too ^^

**Comment by typicalsmc** [comment]
the item not working could be maybe you are using optifine CIT ? If you are using ItemsAdder, please use ItemsAdder directly to add the item instead of Optifine CIT
